src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -25,33 +36,52 @@ export default {
     target_dataset_tooltip: T(''),
 
     recursive_placeholder: T('Recursive'),
-    recursive_tooltip: T(''),
+    recursive_tooltip: T('Replicate all child dataset snapshots.'),
```

Comment:
Not really clear. Verify, but it should work this way: if recursive is not set, replicate just this snapshot. If set, replicate this snapshot and all snapshots it contains.